test/parallel/test-child-process-exec-abortcontroller-promisified.js

```diff
@@ -11,7 +11,8 @@ const invalidArgTypeError = {
 };
 
 const waitCommand = common.isWindows ?
-  `${process.execPath} -e "setInterval(()=>{}, 99)"` :
+  // `"` is forbidden for Windows paths, no need for escaping.
+  `"${process.execPath}" -e "setInterval(()=>{}, 99)"` :
```

Comment:
I love https://docs.python.org/3.12/library/shlex.html#shlex.quote

Reply:
yeah, this just reminds me of such goodies in Python. Not suggesting we should do anything in this comment...